test/trusted_resources_test.go

```diff
@@ -0,0 +1,375 @@
+//go:build e2e
+// +build e2e
+
+/*
+Copyright 2022 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	"context"
+	"crypto"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/sigstore/sigstore/pkg/signature"
+	"github.com/tektoncd/pipeline/pkg/apis/config"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"github.com/tektoncd/pipeline/pkg/pod"
+	"github.com/tektoncd/pipeline/test/parse"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/pkg/apis"
+	"knative.dev/pkg/system"
+	knativetest "knative.dev/pkg/test"
+	"knative.dev/pkg/test/helpers"
+)
+
+var (
+	neededFeatureFlags = map[string]string{
+		"resource-verification-mode": "enforce",
+		// Make sure this is running under alpha integration tests
+		"enable-api-fields": "alpha",
+	}
+	privKey  = "trustedresources-keys/cosign.key"
+	pubKey   = "trustedresources-keys/cosign.pub"
+	password = "1234"
+)
+
+func init() {
+	os.Setenv("PRIVATE_PASSWORD", password)
```

Comment:
Is this environment variable just for the test or part of the feature? Either way could we document it somewhere?

Reply:
yes it is only for this feature, I will add comment for it

Reply:
added comment